### ISSUE TITLE
September should be abbreviated as Sep.

### ISSUE THREAD

**Issue by marlonbernardes** [moment/moment#2873]
,*CONTEXT*
September used to be abbreviated as Sep, but after #2728 it was changed to Sept.

While I agree that in American English Sept. is more common, Sep is understandable and I think we should use it when formatting dates, to keep consistency with other languages (and JavaScript itself).

Some points in favor of changing it back to Sep:

- This is inconsistent with most programming languages: Java, Ruby, Scala and Python (the languages I tested on) all use Sep instead of Sept when formatting dates (snippets below).

- Ruby, Java and Python do not accept Sept as an argument when parsing dates. 
- This could cause issues in the interface, as well: suppose you're displaying the date on an element with a fixed width. September is the only month which requires more space, as it is the only month with 4 characters.

Ruby
````ruby
  Date.strptime("Sep-01-2015", "%b-%d-%Y") // works
  Date.strptime("Sept-01-2015", "%b-%d-%Y") // error: ArgumentError: invalid date
```

Python
```python
datetime.datetime.strptime( "Sep", "%b" ) // works
datetime.datetime.strptime( "Sept", "%b" ) // error:
// Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/_strptime.py", line 328, in _strptime
    data_string[found.end():])
ValueError: unconverted data remains: t
```

JavaScript

```js
var d = new Date(2016, 8, 1);
d.toString() // Thu Sep 01 2016 00:00:00;
```
JavaScript dateformat (most popular library for formatting date on npm - https://www.npmjs.com/package/dateformat)

```js
var date = new Date(2015, 8, 10); 
dateFormat(date, "mmm") // Sep;
```

Java (including its new Time API):
```java
DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMM-dd-yyyy");
LocalDate date = LocalDate.parse("Sep-10-2016", formatter); // works
LocalDate date = LocalDate.parse("Sept-10-2016", formatter); // error:

Exception in thread "main" java.time.format.DateTimeParseException: Text 'Sept-10-2016' could not be parsed at index 3
```

In fact, I discovered this because I faced an error when trying to consume an API that used the pattern `MMM-DD-YYYY` for dates, but failed to accept `Sept` as an argument.

**Comment by marlonbernardes** [comment]
This could be marked as a duplicate of #2854, I think. I didn't see it when I opened this issue.

**Comment by mrlannigan** [comment]
Great explanation of this breaking change.

**Comment by ichernev** [comment]
I agree with you. I didn't pay enough attention to this change. I know a lot of people use moment formats to pass strings between languages / platforms etc. I'll revert it in `2.11.1`. Sorry for the trouble.

Close in favor of #2881.

**Comment by mj1856** [comment]
Yes, indeed.  Apologies for the breaking change.
